modules/mcuboot/hooks/nrf53_hooks.c

```diff
@@ -19,9 +19,51 @@
 #define NET_CORE_VIRTUAL_PRIMARY_SLOT 3
 
 #include <dfu/pcd.h>
+#include <fw_info_bare.h>
 
-int boot_read_image_header_hook(int img_index, int slot,
-		struct image_header *img_head)
+/**
+ * Compare image version numbers
+ * @retval -1           If ver1 is less than ver2.
+ * @retval 0            If the image version numbers are equal.
+ * @retval 1            If ver1 is greater than ver2.
+ */
+int boot_version_cmp_net(const struct flash_area *fap, struct image_header *hdr)
+{
+	int err;
+	const struct fw_info *firmware_info;
+	uint32_t version = 0;
```

Comment:
Don't we need a double word? Ref. the doc of the pcd version read function

Reply:
DWORD (32 bits/4 bytes)

Reply:
I have never heard that term, and googling it seems to indicate that its a Microsoft/windows specific term, should we instead state that its a word? or a 32bit or 4 byte value?

Reply:
Agreed lets change it's obscure and misleading to call it double word.

Reply:
Removed